Resources/Prototypes/_Moffstation/PersonalItems/Plushies/cooksplushie.yml

```
@@ -0,0 +1,25 @@
+# Player: huaqas - Character: Cooks-The-Meth
+- type: entity
+  parent: PlushieLizard
+  id: PersonalItemCooksPlushie
+  name: cooks plushie
```

```suggestion
  name: Cooks plushie
```

We're going with "proper nouns get capitalized in item names".